runtime/autoload/man.vim

```diff
@@ -70,12 +72,14 @@ endfunction
 function! man#read_page(ref) abort
   try
     let [sect, name] = man#extract_sect_and_name_ref(a:ref)
-    let [b:man_sect, name, path] = s:verify_exists(sect, name)
+    let [sect, name, path] = s:verify_exists(sect, name)
     let page = s:get_page(path)
   catch
-    " call to s:error() is unnecessary
+    " call to s:error() does nothing because we were
+    " called from the BufReadCmd autocmd.
```

Comment:
How about the following?

```
    " We do not call s:error because we were called from the
    " BufReadCmd autocmd and thus s:error would do nothing.
```